notanorm/sqlite.py

```diff
@@ -93,13 +97,25 @@ def query_gen(self, sql: str, *args, factory=None):
                 if self.__in_gen[ident] == 0:
                     # Memory cleanup
                     del self.__in_gen[ident]
+                    accum = self.__accum.pop(ident, ())
+                    for a, k in accum:
+                        self.execute(*a, **k)
```

Comment:
this is actually covered.   if you run it locally it works fine.   i don't know why it doesn't work in the runner.   possibly need to upgrade coverage, pytest, pytest-cov?

Reply:
no idea whats up, you can see it hits 98% in the test run, but codecov shows 78%